app/src/main/AndroidManifest.xml

```diff
@@ -5,9 +5,23 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
+        <activity android:name="lt.vilnius.tvarkau.StartActivity" />
+
+        <!-- This strangely named alias is here for backwards compatibility.
+             Renaming activity with LAUNCHER category means that shortcuts created by users
+             on their phones with previous version of the app will disappear or stop working. -->
+        <activity-alias
+            android:name="apps.factory.emiestas.MainActivity"
```

Comment:
BTW, this is not a blocker for merge. Just a minor issue to discuss while the process is moving on. [Here](https://github.com/vilnius/tvarkau-vilniu/pull/8#issuecomment-157017925) @RenatBuga probably wanted to reply in this thread. I agree with focus. But this change is better now or later will be worse (and probably not worth it, so - now or never).